README.rst

```diff
@@ -776,6 +776,31 @@ model, then configure the setting as following:
 This setting allows to disable creating `notification preferences <#notification-preferences>`_
 on running migrations.
 
+``OPENWISP_NOTIFICATIONS_NOTIFICATION_STORM_PREVENTION``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+A ``dict`` containing configuration for notification storm prevention mechanism.
```

Comment:
I think we have to explain what happens, eg:

When the system starts creating a lot of notifications because of a general network outage (eg: a black out), the notification storm prevention mechanism avoids to constantly display many notifications and ring the notification sound. 
The following configuration setting allows to tweak how this mechanism works, if needed.

Feel free to edit/improve.